src/patternfly/components/Drawer/drawer.scss

```diff
@@ -33,58 +43,174 @@ $pf-c-drawer__panel--width-list: (25, 33, 50, 66, 75);
   --pf-c-drawer__panel-body--md--PaddingLeft: var(--pf-global--spacer--lg);
 
   @media screen and (min-width: $pf-global--breakpoint--md) {
+    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--md--FlexBasis);
     --pf-c-drawer__panel-body--PaddingTop: var(--pf-c-drawer__panel-body--md--PaddingTop);
     --pf-c-drawer__panel-body--PaddingRight: var(--pf-c-drawer__panel-body--md--PaddingRight);
     --pf-c-drawer__panel-body--PaddingBottom: var(--pf-c-drawer__panel-body--md--PaddingBottom);
     --pf-c-drawer__panel-body--PaddingLeft: var(--pf-c-drawer__panel-body--md--PaddingLeft);
+    --pf-c-drawer__content-body--m-padding--PaddingTop: var(--pf-c-drawer__content-body--m-padding--md--PaddingTop);
+    --pf-c-drawer__content-body--m-padding--PaddingRight: var(--pf-c-drawer__content-body--m-padding--md--PaddingRight);
+    --pf-c-drawer__content-body--m-padding--PaddingBottom: var(--pf-c-drawer__content-body--m-padding--md--PaddingBottom);
+    --pf-c-drawer__content-body--m-padding--PaddingLeft: var(--pf-c-drawer__content-body--m-padding--md--PaddingLeft);
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--xl--FlexBasis);
+    --pf-c-drawer__panel--MinWidth: var(--pf-c-drawer__panel--xl--MinWidth);
   }
 
   // Box shadow
   --pf-c-drawer__panel--BoxShadow: none;
-  --pf-c-drawer--m-expanded__panel--BoxShadow: var(--pf-global--BoxShadow--md-left);
-  --pf-c-drawer--m-expanded--m-panel-left__panel--BoxShadow: var(--pf-global--BoxShadow--md-right);
+  --pf-c-drawer--m-expanded__panel--BoxShadow: var(--pf-global--BoxShadow--lg-left);
+  --pf-c-drawer--m-expanded--m-panel-left__panel--BoxShadow: var(--pf-global--BoxShadow--lg-right);
 
   // Divider
-  --pf-c-drawer--m-divider__panel--after--Width: var(--pf-global--BorderWidth--sm);
-  --pf-c-drawer--m-divider--m-expanded__panel--after--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-drawer__panel--after--Width: var(--pf-global--BorderWidth--sm);
+  --pf-c-drawer__panel--after--BackgroundColor: transparent;
+  --pf-c-drawer--m-inline--m-expanded__panel--after--BackgroundColor: var(--pf-global--BorderColor--100);
+
+  // Simple list
+  --pf-c-drawer--c-simple-list__title--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__title--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__title--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-drawer--c-simple-list__title--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // Simple list item
+  --pf-c-drawer--c-simple-list__item-link--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__item-link--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // Action
+  --pf-c-drawer__action--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-drawer__panel-body--c-drawer__action--MarginRight: calc(var(--pf-global--spacer--md) * -1);
```

Comment:
I wonder if we should even offer this button if we aren't going to position it for them? I wonder if we should do something like the card - when it has an action menu you use `.pf-c-card__head` with `.pf-c-card__main` and `.pf-c-card__action`, and you put whatever you want to show up on the left in `__main`. And if you have nothing to put beside the close button, maybe you just leave it blank like this example

<img width="859" alt="Screen Shot 2020-02-07 at 4 23 35 PM" src="https://user-images.githubusercontent.com/35148959/74070272-3fbe2680-49c6-11ea-83cb-b5fff910b8a1.png">

Reply:
I expect a close button to be part of the React implementation, so I'll leave for the demo and remove for the component.